File: app/src/main/java/com/akay/fitnass/viewmodel/MainViewModel.java
package com.akay.fitnass.viewmodel;

import android.arch.lifecycle.LiveData;

import com.akay.fitnass.data.model.Runs;

import java.util.List;

public class MainViewModel extends BaseViewModel {
    private LiveData<List<Runs>> mRunsList;

    public MainViewModel() {
        mRunsList = getRepo().getLiveRuns();
    }

    public LiveData<List<Runs>> getLiveRunsList() {
        return mRunsList;
    }

}

File: app/src/main/java/com/akay/fitnass/viewmodel/MainViewModel.kt
package com.akay.fitnass.viewmodel

import androidx.lifecycle.LiveData

import com.akay.fitnass.data.model.Runs

class MainViewModel : BaseViewModel() {
    fun getLiveRunsList(): LiveData<List<Runs>> = repo.getLiveRuns()

    fun deleteDayRuns(runs: Runs) {
        repo.deleteRuns(runs)
    }
}
